docs/usage/reconfigure-renovate.md

```diff
@@ -1,22 +1,41 @@
 ---
 title: Reconfiguring Renovate
```

Comment:
The `title: Reconfiguring Renovate` and the filename `docs/usage/reconfigure-renovate.md` don't  match, do we want to fix that?

Reply:
I don't think it's necessary